lib/message_sender.rb

```diff
@@ -0,0 +1,39 @@
+class MessageSender
+
+	attr_accessor :sms_message
+	attr_reader :order_time, :delivery_time
+
+	def set_order_time(time)
+		@order_time = time.to_s.slice(11,5)
+	end
+
+	def order_time
+		@order_time
+	end
+
+	def delivery_time
```

Comment:
not sure what this is doing or why you need it - again don't do anything not requested in brief